Talkable/Talkable.m

```diff
@@ -824,9 +824,10 @@ - (NSString*)userAgent {
     if (!_userAgent) {
         NSString* userAgent = [self originalUserAgent];
         if (!userAgent) {
-            UIWebView*  webView = [[UIWebView alloc] init];
-            [webView loadHTMLString:@"<html></html>" baseURL:nil];
-            userAgent = [webView stringByEvaluatingJavaScriptFromString:@"navigator.userAgent"];
+            NSString* systemName = [[UIDevice currentDevice] model];
+            NSString* systemVersion = [[UIDevice currentDevice] systemVersion];
+            systemVersion = [systemVersion stringByReplacingOccurrencesOfString:@"." withString:@"_"];
+            userAgent = [NSString stringWithFormat:@"%@ OS %@ webview", systemName, systemVersion];
         }
         if (userAgent) {
```

Comment:
Is this `if` still needed? Looks like the user agent is guaranteed to be present.